Lib/tkinter/__init__.py

```diff
@@ -3535,22 +3539,22 @@ def __init__(self, master=None, cnf={}, **kw):
 
         STANDARD OPTIONS
 
-            background, borderwidth, cursor,
-            exportselection, font, foreground,
-            highlightbackground, highlightcolor,
-            highlightthickness, insertbackground,
-            insertborderwidth, insertofftime,
-            insertontime, insertwidth, padx, pady,
-            relief, selectbackground,
+            background, blockcursor, borderwidth,
```

Comment:
The list of standard options was correct.  The 6 additions
blockcursor, endline (spelled correctly), inactiveselectbackground, insertunfocused, startline, tabstyle,
 should be moved to WIDGET-SPECIFIC OPTIONS